content/articles/building-model-using-code-first/index.md

```diff
@@ -604,11 +607,11 @@ With this, let us run this migration on the database.
 
 ### Adding new classes.
 
-In visual studio, we are going to create a new class called `Category`. We will assign each course to a category. E.g. web development, mobile development, programming language, etc. 
+In visual studio, we are going to create a new class called `Category.cs`. We will assign each course to a category. E.g. web development, mobile development, programming language, etc. 
```

Comment:
You could shorten "visual studio" to "Vs code"

Reply:
I am using visual studio and not visual studio code